README.md

```diff
@@ -4,20 +4,36 @@ Enables OIOs to view and manage screens that provide transit info to riders.
 
 ## Prerequisites
 
-Screenplay requires Postgres. If you don't already have Postgres installed, and you're on a Mac, [Postgres.app](https://postgresapp.com/downloads.html) is an easy way to get started. However, any Postgres instance to which you can connect and in which you have sufficient privileges should work.
+### Postgres
 
-## Development
+If you don't already have Postgres installed, and you're on a Mac, [Postgres.app](https://postgresapp.com/downloads.html) is an easy way to get started. However, any Postgres instance to which you can connect and in which you have sufficient privileges should work.
 
-## Setup
+### Screens
 
-#### Install tools
+Screenplay requires the Screens config to be available at `../screens/priv/local.json`. To generate this file, follow these steps:
+
+1. `cd ..`
+1. `git clone git@github.com:mbta/screens.git`
+1. `cd screens`
+1. Follow the [Getting Started docs](https://github.com/mbta/screens/blob/main/docs/getting_started.md#getting-started) to generate the files needed by Screenplay.
+
+### Realtime-Signs
+
+This repo expects Realtime Signs to be cloned and for `realtime_signs/priv/signs.json` to be available. This can be done by doing the following:
```

Comment:
Think this should be `../realtime_signs`

Reply:
Thanks!

```suggestion
This repo expects Realtime Signs to be cloned and for `../realtime_signs/priv/signs.json` to be available. This can be done by doing the following:
```